src/adapters/hasura/altUser.adapter.ts

```diff
@@ -121,7 +121,7 @@ export class ALTHasuraUserService {
 
     const userSchema = new UserDto(userDto, true);
 
-    console.log("userSchema", userSchema)
+    console.log("userSchema", userSchema);
```

Comment:
remove

Reply:
the userSchema console.log has been present in it since the beginning so I don't think it is okay to just remove it as it gives clarity while creating a user in the alt 
p.s. it is used for alt